### ISSUE TITLE
[Consistency] (mil info page) User encounters design components that are inconsistent or confusing.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#75211]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Veteran Status
**Product:** Veteran Status
**Feature:** MVP Page

## Findings details
**VA.gov Experience Standard - issue:** User encounters design components that are inconsistent or confusing.
**VA.gov Experience Standard - category:** Consistency
**Launch-blocking:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility), @allison0034 (Design)

### Description

**Accessibility concerns:** The "Request your military records (DD214)" heading doesn't fit seems to have the wrong heading level (H3), implying it's a sub-section beneath either "Period of Service" (when service periods are available) or "We can't match your information..." (when periods aren't available), which are both H2s.

Screen reader users often navigate by heading, where their screen reader announces all of the headings on the page and the heading hierarchy. Incorrect heading levels imply incorrect relationships between different sections, which can send users down the wrong path or lead to confusion about what to do next.

In this case, the page structure is straightforward enough that I would expect most users to figure it out without much trouble. But it's a small area of friction that can be smoothed out.

**Design concerns:** Per our guidance, featured content should be used to highlight the most important information, however this is not the most important information on the page.

## Recommended action
**Accessibility:** From my understanding of the content, I think "Request your military records (DD214)" should be an H2. It's a section of content that stands alone, rather than a sub-section within one of the other parts of the page.

**Design:** Remove the featured content UI, if you would like to draw attention to the link use a secondary action link.

## References

- **WCAG Success Criteria:** [WCAG 1.3.1 Info and Relationships - Level A](https://www.w3.org/TR/WCAG21/#info-and-relationships)
- **Modality:** Screen reader
- **Design System Component:** [Featured content ](https://design.va.gov/components/featured-content)
- **Design System Pattern:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by shiragoodman** [comment]
@mtcA6 @Samara-Strauss FYSA per Staging Review on 2/2/2024

**Comment by ACParker89** [comment]
When completed, reassign to Adam for FE work to be completed

**Comment by mtcA6** [comment]
@pamela-drouin were you able to work through this ticket?

**Comment by pamela-drouin** [comment]
@mtcA6 I have [three different treatments](https://www.figma.com/file/zb5ecY9yMnupiLjaH9UmSc/Profile---Military-Information?type=design&node-id=104-996&mode=design) to address the issues in this ticket. None of them are optimal, the page still feels quite hodge-podge given the variety of components on the screen. Profile would really benefit with a holistic make-over since there are so many inconsistencies across the profile sections. Not sure where you want to go with this.

**Comment by mtcA6** [comment]
@pamela-drouin what did CAIA think of the treatments?

**Comment by pamela-drouin** [comment]
@mtcA6 I just incorporated feedback thanks to you and Laura -- check out [treatment 4](https://www.figma.com/file/zb5ecY9yMnupiLjaH9UmSc/Profile---Military-Information?type=design&node-id=104-996&mode=design).

**Comment by mtcA6** [comment]
Looks good to me, and Laura seemed on board w/that treatment so I think we've got a design to move forward with.

**Comment by mtcA6** [comment]
@adamwhitlock1 I think we can't close this ticket until we deploy the changes, are you okay with using this ticket and [treatment 4](https://www.figma.com/file/zb5ecY9yMnupiLjaH9UmSc/Profile---Military-Information?type=design&node-id=203-1685&mode=design&t=CYF2e424kGHhkkeC-4) for the FE work?

**Comment by adamwhitlock1** [comment]
yep, that sounds good to me

**Comment by adamwhitlock1** [comment]
Yep, so the newest changes have been deployed. They are working on a web-component version of the 'action link' and until that is released we are going with the slightly different link style for now, which looks pretty similar so should be fine. You can validate this now.

<img width="906" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBem5wQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7b4533fc44a622af9546ac266830d951cc2dd52c/Screenshot%202024-05-16%20at%209.00.37%E2%80%AFAM.png" alt="Screenshot 2024-05-16 at 9.00.37 AM.png" />
